go/tao/root_host.go

```diff
@@ -140,3 +143,56 @@ func (t *RootHost) RemovedHostedProgram(childSubprin auth.SubPrin) error {
 func (t *RootHost) HostName() auth.Prin {
 	return t.taoHostName
 }
+
+func (s *RootHost) InitCounter(label string, c int64) (error) {
+	fmt.Printf("RootHost.InitCounter %d\n", softtao_counter) //REMOVE
+	softtao_counter = c
+	return nil
+}
+
+func (s *RootHost) GetCounter(label string) (int64, error) {
+	fmt.Printf("RootHost.GetCounter %d\n", softtao_counter) //REMOVE
+	return softtao_counter, nil
+}
+
+func (s *RootHost) RollbackProtectedSeal(label string, data []byte, policy string) ([]byte, error) {
+	fmt.Printf("RootHost.RollbackProtectedSeal\n") //REMOVE
+	softtao_counter = softtao_counter + 1
```

Comment:
Why not add softtao_counter in the root_host struct rather than as a global variable in soft_tao.go?
It looks like root_host.go does not otherwise depend on soft_tao.go (which seems to be used only for unit tests)